Class-10-(Thymeleaf-views).md

```diff
@@ -0,0 +1,37 @@
+# Class 10: Thymeleaf and Views
+
+## In the prep work this week, the students learned
```

Comment:
They also learned how to link CSS files/ other static resources and add photos and things!

Reply:
Added.  Thanks.